src/vstest.console/Resources/Resources.resx

```diff
@@ -730,4 +728,7 @@
   <data name="ProgressIndicatorString" xml:space="preserve">
     <value>Test run in progress</value>
   </data>
+  <data name="TestSourcesDiscovered" xml:space="preserve">
+    <value>A total of {0} test source files are matched with the specified files/pattern.</value>
```

Comment:
A total of {0} test files matched the specified pattern.

Reply:
Done.